src/linker/Linker.Steps/LinkAttributesStep.cs

```diff
@@ -126,18 +129,21 @@ ArrayBuilder<string> GetAttributeChildren (XPathNodeIterator iterator)
 			return children;
 		}
 
-		public void ParseXml (string document)
+		protected override void Process ()
 		{
-			_xmlDocumentLocation = document;
-			_document = new XPathDocument (_xmlDocumentLocation);
+			if (_resource != null && Context.StripLinkAttributes)
```

Comment:
Use single if (_resource != null) check

Reply:
We have the same pattern in BodySubstituterStep - can you please change it there as well?

Reply:
Changed the pattern in LinkAttributesStep, BodySubstituterStep and ResolveFromXmlStep